docs/loading/RoutineLoad.md

```diff
@@ -268,6 +268,128 @@ After submitting the load job, you can execute the [SHOW ROUTINE LOAD](../sql-re
     >
     > You do not need to specify the `COLUMNS` parameter if the names and number of the keys in the JSON object completely match those of fields in the StarRocks table.
 
+### Load Avro-format data
+
+Since v3.0.1, StarRocks supports loading Avro data by using Routine Load.
+
+#### Prepare a dataset
+
+**Avro schema**
+
+1. Create the following  Avro schema file `avro_schema.avsc`:
+
+      ```JSON
+      {
+          "type": "record",
+          "name": "sensor_log",
+          "fields" : [
+              {"name": "id", "type": "long"},
+              {"name": "name", "type": "string"},
+              {"name": "checked", "type" : "boolean"},
+              {"name": "data", "type": "double"},
+              {"name": "sensor_type", "type": {"type": "enum", "name": "sensor_type_enum", "symbols" : ["TEMPERATURE", "HUMIDITY", "AIR-PRESSURE"]}}  
+          ]
+      }
+      ```
+
+2. Register the Avro schema in the [Schema Registry](https://docs.confluent.io/platform/current/schema-registry/index.html).
+
+**Avro data**
+
+Prepare the Avro data and send it to the Kafka topic `topic_0`.
+
+#### Create a table
+
+According to the fields of Avro data, create a table `sensor_log` in the target database `sensor` in the StarRocks cluster. The column names of the table must match the field names in the Avro data. For the mapping of data types between the table columns and the Avro data fields, see [Data types mapping](#Data types mapping).
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/2eeb0804-5d22-4d4e-8e43-156f2a612ad1)
suggest changing it to "the data type mapping"